tutorial/source/index.rst

```diff
@@ -3,92 +3,202 @@
+Making use of this in your own work will require careful reading of :doc:`our overview and programming guide <enumeration>` that opens this series.
+
+Another feature of Pyro is its programmability, the subject of a series of tutorials in :ref:`customizing-inference`.
+Users working with large models where only part of the model needs special attention
+may be interested in `pyro.contrib.easyguide <http://docs.pyro.ai/en/dev/contrib.easyguide.html>`_, introduced in :doc:`the first tutorial of the series <easyguide>`.
+Meanwhile, machine learning researchers interested in developing variational inference algorithms may wish to peruse
+:doc:`the guide to implementing custom variational objectives <custom_objectives>`,
+and a companion example that walks through :doc:`implementing "Boosting BBVI" <boosting_bbvi>`.
+
+Particularly enthusiastic users and potential contributors, especially those interested in contributing to Pyro's core components,
+may even be interested in how Pyro itself works under the hood, partially described in the series :ref:`understanding-pyros-internals`.
+The :doc:`mini-pyro example <minipyro>` contains a complete and heavily commented implementation of a small version of the Pyro language in just a few hundred lines of code,
+and should serve as a more digestable introduction to the real thing.
+
+Tools for specific problems
+-----------------------------
+Pyro is a mature piece of open-source software, not a just science project,
+and in addition to the core machinery for modelling and inference,
+it includes quite a bit of dedicated domain- or problem-specific modelling functionality.
+
+One particular area of strength is time-series modelling via `pyro.contrib.forecasting <http://docs.pyro.ai/en/dev/contrib.forecast.html>`_,
+a library for scaling hierarchical, fully Bayesian models of multivariate time series to thousands or millions of series and datapoints.
+This is described in the series :ref:`time-series`.
+
+Another area of strength is probabilistic machine learning with Gaussian processes.
+`pyro.contrib.gp <http://docs.pyro.ai/en/dev/contrib.gp.html>`_, described in the series :ref:`gaussian-processes`,
+is a library within Pyro implementing a variety of exact or approximate Gaussian process models compatible with Pyro's inference engines.
+Pyro is also fully compatible with `GPyTorch <https://gpytorch.ai/>`_, a dedicated library for scalable GPs,
+as described in `their Pyro example series <https://github.com/cornellius-gp/gpytorch/tree/master/examples/07_Pyro_Integration>`_.
+
+List of Tutorials
+==================
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Introduction:
+   :maxdepth: 1
+   :caption: Introductory Tutorials
+   :name: introductory-tutorials
 
    intro_part_i
    intro_part_ii
    svi_part_i
    svi_part_ii
    svi_part_iii
-   tensor_shapes
-   mle_map
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Advanced:
+   :maxdepth: 1
+   :caption: Hands On with Pyro and PyTorch
```

Comment:
avoid idiomatic english. how about "Advanced Pyro and PyTorch"?

Reply:
I'm trying to avoid calling these "advanced" since that might dissuade potential readers.